### ISSUE TITLE
Inconsistent error while initializing the notion client

### ISSUE THREAD

**Issue by yanmendes** [jamalex/notion-py#269]
I had a script that was working fine up until now and it just started breaking for some reason. Sometimes, when I'm initializing the notion client, I get the following error:

## Initialization
```
notionClient = NotionClient(token_v2=os.environ.get('NOTION_TOKEN'))
```

## Error
```
Traceback (most recent call last):
  File "/Users/<username>/.config/yarn/global/node_modules/serverless/lib/plugins/aws/invokeLocal/invoke.py", line 72, in <module>

    module = import_module(args.handler_path.replace('/', '.'))
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked

  File "<frozen importlib._bootstrap_external>", line 783, in exec_module

  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "./src/handlers/scrapeMentors.py", line 6, in <module>

    from ..services.notion import addMentorToNotion, updateMentorOnNotion, deleteRow
  File "./src/services/notion.py", line 4, in <module>

    notionClient = NotionClient(token_v2=os.environ.get('NOTION_TOKEN'))
  File "/Users/<username>/Library/Python/3.8/lib/python/site-packages/notion/client.py", line 68, in __init__

    self._update_user_info()
  File "/Users/<username>/Library/Python/3.8/lib/python/site-packages/notion/client.py", line 75, in _update_user_info

    self._store.store_recordmap(records)
  File "/Users/<username>/Library/Python/3.8/lib/python/site-packages/notion/store.py", line 292, in store_recordmap

    for id, record in records.items():
AttributeError: 'NoneType' object has no attribute 'items'
```

## Investigation

On [store.py#L292](https://github.com/jamalex/notion-py/blob/master/notion/store.py#L292), I noticed a table named `__json__` that has `None` for records

**Comment by jheddings** [comment]
Getting the same error...  I've tried using the published version on PyPI as well as the latest source.  Seems to be an intermittent issue with data being returned by the Notion servers.  It started just this afternoon.

```
  File "/usr/local/lib/python3.8/site-packages/notion/client.py", line 76, in __init__
    self._update_user_info()
  File "/usr/local/lib/python3.8/site-packages/notion/client.py", line 83, in _update_user_info
    self._store.store_recordmap(records)
  File "/usr/local/lib/python3.8/site-packages/notion/store.py", line 292, in store_recordmap
    for id, record in records.items():
AttributeError: 'NoneType' object has no attribute 'items'
```

**Comment by TristanH** [comment]
Just a +1 here that every call we were making to the NotionClient was failing with the same error, and @jheddings patch fixed it :)

**Comment by mym0404** [comment]
same +1

**Comment by julrach** [comment]
getting the same error as well! +1

**Comment by nanpuhaha** [comment]
Same here.. :(

**Comment by shruddha-deepsource** [comment]
Started seeing this error with my script as well. Failures are seen while initializing. 

>>> client = NotionClient(token_v2=os.environ.get('NOTION_TOKEN'))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/shruddha/opt/anaconda3/envs/bootstrap-github/lib/python3.8/site-packages/notion/client.py", line 76, in __init__
    self._update_user_info()
  File "/Users/shruddha/opt/anaconda3/envs/bootstrap-github/lib/python3.8/site-packages/notion/client.py", line 83, in _update_user_info
    self._store.store_recordmap(records)
  File "/Users/shruddha/opt/anaconda3/envs/bootstrap-github/lib/python3.8/site-packages/notion/store.py", line 292, in store_recordmap
    for id, record in records.items():
AttributeError: 'NoneType' object has no attribute 'items'

**Comment by melonfarm1** [comment]
insert code at line 293 in store.py
`if records is None: continue`

It works fine for me

**Comment by icegalaxy** [comment]
> insert code at line 293 in store.py
> `if records is None: continue`
> 
> It works fine for me

Insert code at line 292, just before the second for loop works for me:
`if not records: continue`

**Comment by antiusio** [comment]
> insert code at line 293 in store.py
> `if records is None: continue`
> 
> It works fine for me

You are right.
I modify this and add all code to my repository. If you are new in programming and not understand, install it for you by commend pip install git+https://github.com/antiusio/notion.git. 
it works for me.

**Comment by pfserra** [comment]
> insert code at line 293 in store.py
> `if records is None: continue`
> 
> It works fine for me

In my case, this did not work..

**Comment by jamalex** [comment]
Reviewing and merging, and will cut a release shortly. Thanks for flagging!

**Comment by yudppp** [comment]
Hi jamalex.
I confirmed that this error was fixed in version 0.0.28.
Thank you:)

**Comment by jamalex** [comment]
Glad to hear it!
